omc3/scripts/betabeatsrc_output_converter.py

```diff
@@ -401,14 +401,14 @@ def convert_old_coupling(
         LOGGER.debug(f"Converting F{rdt} file")
         rdt_dfs[rdt] = rdt_dfs[rdt].rename(
             columns={
-                f"F{rdt}W": "AMP",
-                f"FWSTD{i+1}": f"{ERR}AMP",
-                f"Q{rdt}": "PHASE",
-                f"Q{rdt}STD": f"{ERR}PHASE",
-                f"F{rdt}R": "REAL",
-                f"F{rdt}I": "IMAG",
-                f"MDLF{rdt}R": "MDLREAL",
-                f"MDLF{rdt}I": "MDLIMAG"
+                f"F{rdt}W": f"{AMPLITUDE}",
+                f"FWSTD{i+1}": f"{ERR}{AMPLITUDE}",
+                f"Q{rdt}": "{PHASE}",
+                f"Q{rdt}STD": f"{ERR}{PHASE}",
+                f"F{rdt}R": f"{REAL}",
+                f"F{rdt}I": f"{IMAG}",
+                f"MDLF{rdt}R": f"{MDL}{REAL}",
+                f"MDLF{rdt}I": f"{MDL}{IMAG}",
```

Comment:
Why would it be `MEAS{MDL}` everywhere, but here `{MDL}MEAS` ?

Reply:
Because I'm tired :(